client/my-sites/google-my-business/stats/chart.js

```diff
@@ -55,6 +55,19 @@ function transformData( props ) {
 	} );
 }
 
+function createLegendInfo( props ) {
```

Comment:
if we only use `data`, maybe just pass that?

Reply:
We use `dataSeriesInfo` as well here